webhooks-extension/src/components/WebhookCreate/WebhookCreate.scss

```diff
@@ -0,0 +1,185 @@
+.webhook-create {
+
```

Comment:
Just a couple of comments about styling, where possible `rem` is probably preferable over `px`, when it's 0 this can be just on its own. Also, there are some carbon preset colours that could be useful if there are colours we like available

Reply:
some modifications made